src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
```

Comment:
Make sure to include access modifiers for your fields. If a field is intended to be accessed only within its own class, it should be private.